core/controllers/admin.py

```diff
@@ -471,7 +471,7 @@ def _load_dummy_new_structures_data(self):
             topic_1.add_uncategorized_skill_id(skill_id_1)
             topic_1.add_uncategorized_skill_id(skill_id_2)
             topic_1.add_uncategorized_skill_id(skill_id_3)
-            topic_1.add_subtopic(1, 'Dummy Subtopic Title')
+            topic_1.add_subtopic(1, 'Dummy Subtopic Title', 'url-frag')
```

Comment:
```suggestion
            topic_1.add_subtopic(1, 'Dummy Subtopic Title', 'dummysubtopic')
```

Reply:
Lets change it here as this is not a test files an there might be a conflict if we generate this on the local server.